App/Components/KeyboardResponsiveContainer.tsx

```diff
@@ -26,30 +26,36 @@ export default class KeyboardResponsiveContainer extends React.Component<Props,
   }
 
   keyboardWillChangeFrame = (event: any) => {
-    LayoutAnimation.configureNext(LayoutAnimation.create(
-      event.duration,
-      LayoutAnimation.Types[event.easing]
-    ))
+    LayoutAnimation.configureNext({
+      duration: event.duration,
+      update: {
+        type: LayoutAnimation.Types[event.easing]
+      }
+    })
     this.setState({
         height: this.bottomViewY - event.endCoordinates.screenY
     })
   }
 
   keyboardWillAppear = (event: any) => {
-    LayoutAnimation.configureNext(LayoutAnimation.create(
-      event.duration,
-      LayoutAnimation.Types[event.easing]
-    ))
+    LayoutAnimation.configureNext({
+      duration: event.duration,
+      update: {
+        type: LayoutAnimation.Types[event.easing]
+      }
+    })
     this.setState({
         height: this.bottomViewY - event.endCoordinates.screenY
     })
   }
 
   keyboardWillHide = (event: any) => {
-    LayoutAnimation.configureNext(LayoutAnimation.create(
```

Comment:
This `LayoutAnimation.create` was creating `LayoutAnimation` configuration objects that were sometimes considered invalid and causing that crash. Just directly specifying the configuration object like this seems to fix the crash for some reason. Don't totally understand it, but cool!